### PR TITLE
New Docs

### PR DESCRIPTION
**Pull request by AlCap23**

### CODE REVIEW

File: docs/src/index.md [hunk 1]
```diff
@@ -0,0 +1,18 @@
+# DataDrivenDiffEq.jl
+
+DataDrivenDiffEq.jl is a package for estimating equationfree and equationbased models for discrete and continouos differential equations.
```
Comment (ChrisRackauckas) [inline docs/src/index.md:3]:
```suggestion
DataDrivenDiffEq.jl is a package for estimating equation-free and equation-based models for discrete and continuous differential equations.
```

File: docs/make.jl [hunk 2]
```diff
@@ -0,0 +1,37 @@
...
+    sitename="DataDrivenDiffEq.jl",
+    authors="Julius Martensen, Chris Rackauckas",
+    modules=[DataDrivenDiffEq],
+    clean=true,doctest=false,
+    format = Documenter.HTML(#analytics = "UA-90474609-3",
+                             assets = ["assets/favicon.ico"],
+                             canonical="https://docs.sciml.ai/stable/"),
+    pages=[
+        "Home" => "index.md",
+        "basis.md",
```
Comment (ChrisRackauckas) [inline docs/make.jl:13]: basis is more of a deeper reference. You want to grab the reader first, so the methods DMD SInDy etc. should come first, with the details about basis's afterwards for those now curious.

**Comment by ChrisRackauckas** [comment]
@kanav99 can you help get these deploying? I'll get datadriven.sciml.ai setup on the server.

File: docs/src/koopman/dmd.md [hunk 3]
```diff
@@ -0,0 +1,231 @@
...
+
+Additionally, DMD objects can be updated to add new measurements via:
+
+```julia
+update!(dmd, x; Δt = 0.0, threshold = 1e-3)
+```
+
+## Examples
+
+### Structural Identification with SInDy
```
Comment (ChrisRackauckas) [inline docs/src/koopman/dmd.md:66]: these should probably move out to tutorials pages which come before the reference pages.

**Comment by kanav99** [comment]
Sure!

**Comment by AlCap23** [comment]
I think there are still some typos hidden, but the main skeleton is ready.

**Comment by ChrisRackauckas** [comment]
I'll merge this so we can make sure we can get the servers all setup.

**Pull request merged**